src/cmds/qdel.c

```diff
@@ -167,20 +240,33 @@ char **envp;
 		fprintf(stderr, "qdel: unable to initialize security library.\n");
 		exit(1);
 	}
-
+	
+	/* allocate enough memory to store array of server instances */
+	no_of_pbsservers = 10;
+	pbs_servers = (pbs_svr_t *)calloc(no_of_pbsservers, sizeof(pbs_svr_t));
+	if (pbs_servers == NULL) {
+		fprintf(stderr, "qdel: unable to allocate memory.\n");
+		exit(2);
+	}
+		
 	for (; optind < argc; optind++) {
-		int connect;
-		int stat=0;
-		int located = FALSE;
-
 		pbs_strncpy(job_id, argv[optind], sizeof(job_id));
```

Comment:
Can't we argv[optind] directly in get_server()?

Reply:
I will change this.